README.md

```diff
@@ -93,6 +102,72 @@ the algorithm/s:
   * SM4-GCM: -sm4-gcm
   * SM4-CCM: -sm4-ccm
 
+## Testing TLS
+
+SM ciphers are able to be used with TLSv1.2 and TLSv1.3.
+
+Note: SM2, SM3 and at least one SM4 cipher must be built in order for SM
+ciphers suite to work. All algorithms must be SM.
+
+The cipher suites added are:
+  - ECDHE-ECDSA-SM4-CBC-SM3 (TLSv1.2, --enable-sm2 --enable-sm3 --enable-sm4-cbc)
+  - ECDHE-ECDSA-SM4-GCM-SM3 (TLSv1.2, --enable-sm2 --enable-sm3 --enable-sm4-gcm)
+  - ECDHE-ECDSA-SM4-CCM-SM3 (TLSv1.2, --enable-sm2 --enable-sm3 --enable-sm4-ccm)
+  - TLS13-SM4-GCM-SM3 (TLSv1.3, --enable-sm2 --enable-sm3 --enable-sm4-gcm)
+  - TLS13-SM4-CCM-SM3 (TLSv1.3, --enable-sm2 --enable-sm3 --enable-sm4-ccm)
+
+### Example of using SM cipher suites with TLSv1.2
+
+An example of testing a TLSv1.2 cipher suite:
```

Comment:
In response to my long comment. I think changing this line from 
`An example of testing a TLSv1.2 cipher suite:`
to 
`An example of testing a TLSv1.2 with the "ECDHE-ECDSA-SM4-CBC-SM3" cipher suite:`
This should help bring attention to the fact this is specifically testing this suite in the example command. 
I originally read this as "to test TLSv1.2 with your suite" where suite is the enable configuration that you set. This should now hopefully bring a connection cipher suites sections to the example section.

Reply:
fixed